src/js/default-props.js

```diff
@@ -8,3 +8,11 @@ export const defaultProps = {
 export const extendDefaultTheme = theme => {
   defaultProps.theme = deepMerge(base, theme);
 };
+
+/*
+  Pass `theme` for component which can be loacted outside of theme context.
+  To be used as argument for `attrs` method from `styled-components`.
+ */
+export const ehnancePropsWithTheme = props => ({
```

Comment:
Minor spelling change
```suggestion
export const enhancePropsWithTheme = props => ({
```

Reply:
Such a silly thing :) Thanks for pointing, fixed that.